### ISSUE TITLE
Doc Improvement

### ISSUE THREAD

**Issue by SamyAteia** [confident-ai/deepeval#1225]
Maybe add the "expected_output" field to the first test case definition in the quickstart guide: https://docs.confident-ai.com/docs/getting-started#create-your-first-test-case

This would prevent errors when the user later on tries to test custom metrics example code that relies on this field and reuses the first test case instead of copying the test case without text from the custom metrics example.

**Comment by penguine-ip** [comment]
@SamyAteia I didn't want to deter users who feels like LLM evaluation is not for them because they don't have a labelled dataset.

**Comment by SamyAteia** [comment]
True, another solution would be to  just fill in the example for the custom metric so users wouldn't reuse the one from the steps before.

**Comment by penguine-ip** [comment]
@SamyAteia We try to make it as easy as possible - if a user reuses the test case and uses a metric that requires the expected output the error message will tell you you'll need the expected output for the test case: https://github.com/confident-ai/deepeval/blob/main/deepeval/metrics/utils.py#L177

**Comment by SamyAteia** [comment]
I believe this function is not used when following the quickstart to implement the custom rouge metric.
